cupy/core/raw.pyx

```diff
@@ -223,6 +228,26 @@ cdef class RawKernel:
         attr = driver.CU_FUNC_ATTRIBUTE_PREFERRED_SHARED_MEMORY_CARVEOUT
         driver.funcSetAttribute(self.kernel.ptr, attr, fraction)
 
+    def compile(self, options=None, backend=None, log_stream=None):
+        """ Compile the current kernel
+
+        Compiles the kernels in code, and returns a Module instance. If the
+        arguments are not given, the corresponding attributes of the RawKernel
+        instance that were given during initialization are used.
+
+        Args:
+            options (tuple of str): Compiler options passed to the backend (NVRTC
+                or NVCC). For details, see
+                https://docs.nvidia.com/cuda/nvrtc/index.html#group__options or
+                https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#command-option-description
+            backend (str): Either `nvrtc` or `nvcc`. Defaults to `RawKernel.backend`
```

Comment:
Oops, I think we did it wrong before...Could you correct it here as well as in other places?
```suggestion
            backend (str): Either ``'nvrtc'`` or ``'nvcc'``. Defaults to
                :attr:`RawKernel.backend`.
```